fragdenstaat_de/fds_blog/templates/fds_blog/includes/_authors.html

```diff
@@ -6,13 +6,21 @@
         <div class="blog-author-images" aria-hidden="true">
             {% for author in authors %}
                 {% if author.user.profile_photo %}
-                    <a href="{{ author.get_absolute_url }}" tabindex="-1">
+                    {% if author.get_absolute_url %}
```

Comment:
The `author.user.profile_photo` check is sufficient. If a user has a profile photo, they also have a URL, see:
https://github.com/okfde/fragdenstaat_de/blob/dfc00193a74585fcafd3fc54a752eabdaa93797f/fragdenstaat_de/fds_blog/models.py#L130